### PR TITLE
fix: Conda + Tensorflow example

### PR DESCRIPTION
**Pull request by thundergolfer**
Our TF example has been failing for just over a week. These changes fix it for me when I run it locally.

### CODE REVIEW

File: 06_gpu/tensorflow_tutorial.py [hunk 1]
```diff
@@ -25,7 +25,9 @@
 stub = modal.Stub(
     "example-tensorflow-tutorial",
     image=(
-        modal.Image.conda().conda_install(["cudatoolkit=11.2", "cudnn=8.1.0"]).pip_install(["tensorflow", "pathlib"])
+        modal.Image.conda()
+        .conda_install(["cudatoolkit=11.2", "cudnn=8.1.0", "cuda-nvcc"], channels=["conda-forge", "nvidia"])
+        .pip_install(["tensorflow~=2.9.1"])
```
Comment (thundergolfer) [inline 06_gpu/tensorflow_tutorial.py:30]: Later versions of Tensorflow (2.10.x, 2.11.x), become more difficult to install with Conda: https://github.com/tensorflow/tensorflow/issues/56927.

File: 06_gpu/tensorflow_tutorial.py [hunk 2]
```diff
@@ -25,7 +25,9 @@
 stub = modal.Stub(
     "example-tensorflow-tutorial",
     image=(
-        modal.Image.conda().conda_install(["cudatoolkit=11.2", "cudnn=8.1.0"]).pip_install(["tensorflow", "pathlib"])
+        modal.Image.conda()
+        .conda_install(["cudatoolkit=11.2", "cudnn=8.1.0", "cuda-nvcc"], channels=["conda-forge", "nvidia"])
```
Comment (thundergolfer) [inline 06_gpu/tensorflow_tutorial.py:29]: Installing `cuda-nvcc` removes these 'looking for PTXAS' warnings:

```
2022-12-04 00:50:11.229493: W tensorflow/stream_executor/gpu/redzone_allocator.cc:314] INTERNAL: Failed to launch ptxas
Relying on driver to perform ptx compilation.
Modify $PATH to customize ptxas location.
```

File: 06_gpu/tensorflow_tutorial.py (hunk 1 above)
Comment (erikbern) [reply 06_gpu/tensorflow_tutorial.py:30]: maybe we shouldn't use conda for it?

**Comment by erikbern** [comment]
thanks for fixing!

File: 06_gpu/tensorflow_tutorial.py (hunk 1 above)
Comment (thundergolfer) [reply 06_gpu/tensorflow_tutorial.py:30]: Maybe, but it's the only `conda` example we have I think, and it's helpful to catch problems.

**Pull request merged**